### DISCUSSION TITLE
Publish private products

### DISCUSSION THREAD

**Discussion by mereolog** [edmcouncil/ontology-publisher#88] [Code Improvements]
We need to be able to publish products from a private repository in such a way that access to them is restricted.

**Comment by przemekgradzki** [comment]
Access to the products published by
_ontology-publisher_ depends on the access mechanisms of the server on which the products are published, not the _ontology-publisher_ itself.

**Comment by mereolog** [comment]
> Access to the products published by
> _ontology-publisher_ depends on the access mechanisms of the server on which the products are published, not the _ontology-publisher_ itself.

Fair enough, still we need to set up the access mechanism as above - the fact that this is not the right place to discuss it is less relevant. The issue will be converted to a discussion and left for the audit purposes.
